vitrina/users/models.py

```diff
@@ -149,6 +149,24 @@ def is_representative_of(self, organization: Organization, check_agreement_right
 
         return representatives.exists()
 
+    def is_information_system_representative_for(self, organization: Organization) -> bool:
+        org_type = ContentType.objects.get_for_model(Organization)
+        return Representative.objects.filter(
+            user=self,
+            content_type=org_type,
+            object_id=organization.pk,
+            information_system_representative=True,
+        ).exists()
+
+    def is_open_data_representative_for(self, organization: Organization) -> bool:
+        org_type = ContentType.objects.get_for_model(Organization)
```

Comment:
I'd move this out as a separate method (The content type retrieve from db) and add `@cached_property`, that way, you won't make calls for everytime this function is called.